src/components/Editor/GutterMenu.js

```diff
@@ -139,16 +140,36 @@ class GutterContextMenuComponent extends Component {
     const { contextMenu, ...props } = nextProps;
     const { event } = contextMenu;
     const sourceId = props.selectedSource ? props.selectedSource.id : "";
-    const line = lineAtHeight(props.editor, sourceId, event);
+
+    const classMatch = event.target.className.match(/cl-([0-9]+)-([0-9]+)/);
+    let line = 0;
+    let column = undefined;
+    if (classMatch) {
+      line = classMatch[1];
+      column = classMatch[2];
+    } else {
+      line = lineAtHeight(props.editor, sourceId, event);
+    }
+
     const breakpoint = nextProps.breakpoints.find(
-      bp => bp.location.line === line
+      bp => bp.location.line === line && bp.location.column === column
+    );
```

Comment:
same here, lets try and go off of pause points or visible column breakpoints which the component could have access too.